File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucTinhTrangService.cs
namespace Service.Implement
{
    public class DanhMucTinhTrangService : BaseService<DanhMucTinhTrang, IDanhMucTinhTrangRepository>
    , IDanhMucTinhTrangService
    {
    private readonly IDanhMucTinhTrangRepository _DanhMucTinhTrangRepository;
    public DanhMucTinhTrangService(IDanhMucTinhTrangRepository DanhMucTinhTrangRepository) : base(DanhMucTinhTrangRepository)
    {
    _DanhMucTinhTrangRepository = DanhMucTinhTrangRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucTinhTrang.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucTinhTrang } from 'src/app/shared/DanhMucTinhTrang.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucTinhTrangService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucTinhTrang";
    }
}

